components/Button.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+const Button = styled.button`
+  width: 250px;
+  background: #ffc535;
```

Comment:
Let's use constants for our color palette

Reply:
Okay